_data/social.yml

```diff
@@ -216,4 +216,4 @@ websites:
       img: zocle.png
       tfa: Yes
       sms: Yes
-      doc: http://university.zocle.com/2015/03/30/new-security-feature-two-step-verification/
+      doc: http://university.zocle.com/2015/03/30/new-security-feature-two-step-verification/
```

Comment:
As per our `.editorconfig`, it contains `insert_final_newline = true`.
Please respect that.